manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,14 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
+  ){
+
+  file { "${foreman_plugins_dir}/foreman_gutterball_auxiliary.yaml":
```

Comment:
How about just `foreman_gutterball.yaml` ? I don't see why it needs a special name when it is simply the config file for foreman_gutterball.